app/controllers/contests_controller.rb

```diff
@@ -21,12 +21,20 @@ def create
     end
   end
 
+
+  def invite
+    user = User.find_by(email: params[:email])
+    contest = Contest.find(params[:id])
```

Comment:
three occurences of `@contest = Contest.find(params[:id])`, maybe it's time to create a `before_action :set_contest` ;-)